### ISSUE TITLE
CMakeLists.txt not included in tagged releases

### ISSUE THREAD

**Issue by jasonliu--** [mumble-voip/mumble#5367]
### Description

When downloading any of the source tarballs from the releases section (https://github.com/mumble-voip/mumble/releases), the `CMakeLists.txt` file cannot be found anywhere inside.

### Steps to reproduce

1. Go to https://github.com/mumble-voip/mumble/releases
2. Download any of the source tarballs. I've tried `mumble-1.3.4.tar.gz`, `mumble-1.3.4.zip`, `Source code (zip)`, and `Source code (tar.gz)`.
3. Extract, and try to find the `CMakeLists.txt`. I used the `find` command: `find . -name CMakeLists.txt`, which returns nothing.

### Mumble version

Any. So far I've looked at the 1.3.4 and 1.3.3 tagged releases, but I suspect that all will be the same

### Mumble component

Both

### OS

macOS

### Reproducible?

Yes

### Additional information

I am trying to package mumble for the [MacPorts package management system](https://www.macports.org/). The MacPorts build system very much prefers to fetch the source code from a GitHub tagged release, and we are generally discouraged from simply cloning a commit hash from `master`. Since the build system cannot find a `CMakeLists.txt` file, it's obviously failing to build the package.

**Comment by ZeroAbility** [comment]
1.3.x and older do not use CMake. 1.4.x and newer, do.

**Comment by jasonliu--** [comment]
Then how do I build the source? I don't see a `Makefile`, or any files related to autotools.

**Comment by ZeroAbility** [comment]
`qmake` is the build tool and there is a `.pro` file in the root of the source directory. I'm not able to see the documentation in the [1.3.4](https://github.com/mumble-voip/mumble/tree/1.3.4) tag. It's possible it may have been moved.

**Comment by jasonliu--** [comment]
Ah, ok. MacPorts does have a `qmake` build target that I can use. However, I don't see any `.pro` file in the root of the source?

**Comment by jasonliu--** [comment]
Ah, yes `main.pro`, I see it! I will try to get MacPorts to do the build using `qmake`, thanks!

> I'm not able to see the documentation in the [1.3.4](https://github.com/mumble-voip/mumble/tree/1.3.4) tag. It's possible it may have been moved.

And yes, all of the mumble documentation that I've found online has talked about building using CMake, which is why I was confused. Maybe my googling skills fell short, but I didn't even find any results that talked about moving to cmake from whatever the "old" make system was (which I've now learned from you was qmake).
